api/views.py

```diff
@@ -45,6 +47,15 @@ class ExportInstructorLocationsView(ListAPIView):
     serializer_class = ExportInstructorLocationsSerializer
 
 
+class ExportMembersView(ListAPIView):
+    """Show everyone who qualifies as an SCF member."""
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    paginator = None  # disable pagination
+
+    queryset = get_members()
```

Comment:
I think that:

* it can be fixed by making `get_queryset(self)` method that return `get_members()`
* fixing it will fix all or almost all of the failing tests.